.github/notification.js

```diff
@@ -0,0 +1,23 @@
+const path = require("path");
+const require = createRequire(import.meta.url);
```

Comment:
what is createRequire? this repo isn't defined as a `type: "module"`, so would it be required here?